server/routes/loans.js

```diff
@@ -2,22 +2,25 @@ import express from 'express';
 import Loan from '../controllers/loans';
 import validate from '../middleware/validation';
 import Auth from '../middleware/auth';
+import loans from '../queries/loanQueries';
+import dbvalidate from '../middleware/dbValidations';
 
 const loanRouter = express.Router();
 
+// const {
+//   verifyStatus, verifyId, verifyLoanId, verifyAmount, verifyFields, validateQuery,
+// } = validate;
+// const {
+//   getLoanById, getLoans, getRepaymentById, create, approve, repayment,
+// } = Loan;
```

Comment:
Code that is not needed shouldn't be commented but removed from the codebase.

Reply:
duly noted